### PR TITLE
feat(preview): add preview frame and label

### PR DESCRIPTION
**Pull request by beerecca**
@usabilla/mallard

### CODE REVIEW

File: src/components/_preview.scss [hunk 1]
```diff
@@ -16,8 +16,29 @@
...
+  &__label {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  &__frame {
+    position: absolute;
+    top: sb-px2rems(20px);
```
Comment (keepitterron) [inline src/components/_preview.scss]: why `20px`? isn't our spacing [...16,23...]?

Comment (beerecca) [inline src/components/_preview.scss]: hmm good point... i got this from the grid margin. So maybe that needs to be 23 too..?

Comment (keepitterron) [inline src/components/_preview.scss]: Ah yes,
we should revisit the grid system now that the styleguide is updated and we have one example of usage.
Should we open a ticket for it?

Comment (beerecca) [inline src/components/_preview.scss]: There's one on the styleguide jira project STYLE-40, I just added some notes to it.

**Pull request merged**